ui/src/app/components/webcomponents/action-test-data-runtime-variable-suggestion.component.ts

```diff
@@ -219,4 +219,11 @@ export class ActionTestDataRuntimeVariableSuggestionComponent implements OnInit
     }
   }
 
+  /**
```

Comment:
/**
 * short desc about the block
 * @func|getter|constructor
 * @param {type} prop_name - desc
 */

you can follow this patter for subsequent items @manikanta-reddy-t